source/jormungandr/jormungandr/interfaces/v1/serializer/base.py

```diff
@@ -131,7 +131,7 @@ def to_value(self, value):
 
     @staticmethod
     def _get_all_possible_values(pb_type):
-        return [v.name for v in pb_type.DESCRIPTOR.values]
+        return [v.name.lower() for v in pb_type.DESCRIPTOR.values]
```

Comment:
I think that we keep a few enum in upper case.

Reply:
As we discussed yesterday, among all the enum  types, we have some in uppercase where all the attributs are lowercase (forced in the code). What should i do? manage some in uppercase and some in lowercase, all in lowercase or all in uppercase?